start_garmin_microservice.sh

```
@@ -0,0 +1,77 @@
+    echo "Existing venv is Python $VENV_VER (requires >= 3.10). Recreating venv..."
+    rm -rf venv
+    RECREATE_VENV=true
+  fi
+else
+  RECREATE_VENV=true
+fi
+
+if [ "$RECREATE_VENV" = true ]; then
+  echo "Creating virtual environment with $PYTHON_BIN..."
+  "$PYTHON_BIN" -m venv venv
+  source venv/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)